types/react-autosuggest/react-autosuggest-tests.tsx

```diff
@@ -564,3 +565,38 @@ const test: Autosuggest.InputProps<{ foo: string }> = {
     value: 'foo',
     anything: false // $ExpectError
 };
+
+function testSingleSection() {
+    const suggestions: string[] = [];
+    <Autosuggest
+        suggestions={suggestions}
+
+        // Required
+        getSuggestionValue={suggestion => suggestion}
+        inputProps={{
+            onChange: () => {},
+            value: ''
+        }}
+        onSuggestionsFetchRequested={() => {}}
+        renderSuggestion={suggestion => suggestion}
+    />;
+}
+
+function testMultiSections() {
+    interface Section { title: string; suggestions: string[]; }
+    const sections: Section[] = [];
+    <Autosuggest
+        multiSection
+        suggestions={sections}
+        getSectionSuggestions={section => section.suggestions}
+
+        // Required
+        getSuggestionValue={(suggestion: string) => suggestion}
```

Comment:
Ideally the param annotation wouldn't be needed here, but TS has trouble with inference for some reason. Reduced test case:

```tsx
type Props<Suggestion, Section> = {
    sections: Section[];
    getSectionSuggestions: (section: Section) => Suggestion[];
    getSuggestionValue: (suggestion: Suggestion) => string;
};

declare class Autosuggest<Suggestion, Section> extends React.Component<
    Props<Suggestion, Section>
> {}

// Example usage

type Section = { title: string; suggestions: string[] };
declare const sections: Section[];

<Autosuggest
    sections={sections}
    getSectionSuggestions={section => section.suggestions}
    getSuggestionValue={suggestion => suggestion} // error
    // Workaround:
    // getSuggestionValue={(suggestion: string) => suggestion}
/>;
```